dev_guide/routes.adoc

```diff
@@ -133,6 +133,22 @@ TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
 
+[[creating-labeled-routes]]
+== Creating Routes with Labels
+
+You can create a route based on labels. A explanation how can be found at
+link:../architecture/core_concepts/routes.adoc#router-sharding
```

Comment:
- This is not accurate -- the route is not **based on** labels.
- There is no link text here.
- The URL needs `.html` rather than `.adoc`.

How about we address all three issues, and make things more concise, as well, with:

```asciidoc
link:../architecture/core_concepts/routes.adoc#router-sharding[Router sharding]
requires routes with labels.
```